agent/tool-scripts/base-tool

```diff
@@ -580,18 +580,18 @@ install)
 		local _tpn="${1}"
 		local _tpv="${2}"
 		local _installed_rpm
-		local rc
-    _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
-    rc=${?} 
-		if [[ ${rc} != 0 ]]; then
+		local _rc
+        _installed_rpm="$(require-rpm "${_tpn}" "${_tpv}")"
+        _rc=${?} 
+		if [[ ${_rc} != 0 ]]; then
```

Comment:
Lines 583 and 586 are indented two levels, each indicated by a hard-tab character.  Lines 584 and 585 appear now to be indented two levels, but each level is signified by four blank-space characters.  If you cannot get your editor to use tabs for indentation in this module, you'll need to insert the characters "manually".  (You can cut-and-paste them, if you can't come up with a better way.)

Reply:
i will try with a different editor than `vim`